packages/app/src/router/router.ts

```diff
@@ -0,0 +1,12 @@
+import { createWebHashHistory, createRouter as _createRouter } from 'vue-router'
+import generatedRoutes from 'virtual:generated-pages'
```

Comment:
This is what is causing CI to fail - launchpad is now using this router, and it cannot find these virtual pages.

We need some way to conditionally pass plugins to `cy.mount` - launchpad doesn't need to have a router.

Reply:
Yeah I meant that `packages/app` was failing due to schema stitching... not that `packages/launchpad` was failing because of it.